_posts/en/newsletters/2024-08-30-newsletter.md

```diff
@@ -0,0 +1,152 @@
+  checkpoints are protecting against.
+
+- [Bulletproof++ as generic ZKP ala SNARKs?]({{bse}}119556)
+  Liam Eagen details the type of succinct non-interactive argument of knowledge
+  (SNARKs) currently in use and how bulletproofs, [BitVM][topic acc], and
+  [OP_CAT][topic op_cat] might be used to verify such proofs in Bitcoin Script.
+
+- [How can OP_CAT be used to implement additional covenants?]({{bse}}123829)
+  Brandon - Rearden describes how the proposed OP_CAT opcode could provide
+  [covenant][topic covenants] functionality to Bitcoin scripts.
+
+- [Why do some bech32 bitcoin addresses contain a large number of 'q's?]({{bse}}123902)
+  Vojtěch Strnad reveals that the OLGA protocol encodes arbitrary data into
+  P2WSH outputs with part of the scheme requiring 0-padding (0 is encoded as 'q'
+  in [bech32][topic bech32]).
+
+- [How does a 0-conf signature bond work?]({{bse}}124022)
+  Matt Black outlines how funds locked in an OP_CAT-based covenant could provide
+  incentives for spenders to not [RBF][topic rbf] fee bump their transactions,
+  potentially increasing acceptance of zero-confirmation transactions.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 24.08rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+- [LND v0.18.3-beta.rc1][] is a release candidate for a minor bug fix
+  release of this popular LN node implementation.
+
+- [BDK 1.0.0-beta.2][] is a release candidate for this library for
+  building wallets and other Bitcoin-enabled applications.  The original
+  `bdk` Rust crate has been renamed to `bdk_wallet` and lower layer
+  modules have been extracted into their own crates, including
+  `bdk_chain`, `bdk_electrum`, `bdk_esplora`, and `bdk_bitcoind_rpc`.
+  The `bdk_wallet` crate "is the first version to offer a stable 1.0.0 API."
+
+- [Bitcoin Core 28.0rc1][] is a release candidate for the next major
+  version of the predominant full node implementation.  A [testing
+  guide][bcc testing] is being prepared.
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [LDK #3263][] simplifies how it handles [onion messages][topic onion messages]
+  responses by removing the message type bound from the `ResponseInstruction`
```

Comment:
Just a nit. The title of the PR _“Remove message type bound on ResponseInstruction”_ means _“Remove (message type) bound on ResponseInstruction”_ and not _”Remove (message type bound) on ResponseInstruction.”_ The parameters on structs are usually called type parameters.

```suggestion
  responses by removing the message type parameter from the `ResponseInstruction`
```

Reply:
I dont have a strong opinion on it, but took this suggestion.